app/components/Carousel.tsx

```diff
@@ -115,14 +131,50 @@ export function Carousel(props: CarouselProps) {
             )
           }
 
+          let socialLinks = null
+          if (props.preset === "dynamic") {
+            const { socialLinks: socialLinksData } = item as DynamicCarouselItem
+            socialLinks = socialLinksData && (
+              <View style={$linksContainer}>
+                {socialLinksData.twitter && (
+                  <IconButton
```

Comment:
I think you could probably DRY this up a bit...maybe if you had a little custom component called `SocialButton` which takes URL and icon name as props?

Also you could avoid the three conditional renders by defining a little object like this:  
```
 const socialButtons = [
    { url: socialLinks.twitter, icon: "twitter" },
    { url: socialLinks.github, icon: "github" },
    { url: socialLinks.externalURL, icon: "link" },
  ]
```
then mapping over it and returning your custom component.

Reply:
This is why I need you! 👍🏼 This didn't even come to my mind even though I totally did this in Oxxxx project LOL.